mkt/search/tests/test_views.py

```diff
@@ -796,6 +796,25 @@ def test_no_filter(self):
         res = self.client.get(self.url, data={'filtering': 0})
         eq_(len(res.json['objects']), 1)
 
+    def test_no_filter_excludes_deleted_or_disabled(self):
```

Comment:
This should probably be two different tests. My rubric: if a failure in different parts would be  of a failure of a different code path, it should be housed in a different test.